core/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -3,10 +3,31 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
+  xmlns:tools="http://schemas.android.com/tools"
   android:fitsSystemWindows="true">
 
-  <include layout="@layout/layout_standard_app_bar" />
+  <com.google.android.material.appbar.AppBarLayout
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintTop_toTopOf="parent"
+    android:background="@color/black"
```

Comment:
now that we are specifying the color twice I am less okay with it.

I can't think of a scenario where we don't want an AppBar to be black so using a theme to style this is a good solution. See `toolbarStyle` in `themes.xml` for an example of what I want

Reply:
https://www.youtube.com/watch?v=Owkf8DhAOSo this video is basically the best explanation you will get on themes/styles if you find the topic confusing, which it profoundly is